src/pd/client.rs

```diff
@@ -147,6 +215,13 @@ pub struct RpcClient {
 
 impl RpcClient {
     pub fn new(endpoints: &str) -> Result<RpcClient> {
+        let endpoints: Vec<String> = endpoints.split(',')
+            .map(|s| s.trim().to_owned())
+            .filter(|s| !s.is_empty())
+            .collect();
+
+        try!(validate_endpoints(&endpoints));
```

Comment:
We need to add some retries here, see line 231.
If PD and TiKV start at the same time, TiKV may fail if PD is not ready yet.